src/sequence_align/pairwise.py

```diff
@@ -139,17 +146,64 @@ def hirschberg(
         See https://en.wikipedia.org/wiki/Hirschberg%27s_algorithm for more information.
     """
     # First, map the sequences to integers
-    idx2symbol, seq_a_indices, seq_b_indices = _entry2idx(seq_a, seq_b, gap)
+    idx2symbol, seq_a_indices, seq_b_indices = _entry2idx(seq_a, seq_b, gap=gap)
```

Comment:
gap is still a positional arg in the function signature

Reply:
ah, good catch -- an earlier version of my changes made it a kwarg, but forgot to switch it back